frontend/templates/views/partials/header.j2

```diff
@@ -11,30 +11,144 @@
 <header class="ap--header {% if not second_level %}ap--header-fixed{% endif%}" [class]="mainmenuopen ? 'ap--header {% if not second_level %}ap--header-fixed{% endif%} mainmenuopen' : 'ap--header {% if not second_level %}ap--header-fixed{% endif%}'">
   {% include '/views/partials/banner.j2' %}
 
-  <div class="ap-o-header">
+  <ul class="ap-o-header">
```

Comment:
This element can't be a `<ul>` as it has other children than `<li>`, additionally you didn't change the closing tag.